app/models/manageiq/providers/ansible_tower/configuration_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,47 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::EventCatcher::Stream
+  class ProviderUnreachable < ManageIQ::Providers::BaseManager::EventCatcher::Runner::TemporaryFailure
+  end
+
+  def initialize(ems)
+    @ems = ems
+    @last_activity = nil
+    @stop_polling = false
```

Comment:
because I want to make this read positive
```ruby
throw :stop_polling if @stop_polling
```

vs
```ruby
throw :stop_polling unless @polling_enabled
```